lib/KalturaModel.php

```diff
@@ -236,7 +236,18 @@ public function listPlayersUiConfs() {
 
 		$uiConfs = $this->_client->uiConf->listAction( $filter );
 
-		return $uiConfs;
+
+		// filter out playlist players
+		$players = new stdClass();
+		$players->objects = array();
+
+		foreach( $uiConfs->objects as $uiConf ) {
+			if( strpos($uiConf->tags, 'playlist') === false ) {
```

Comment:
It's on purpose, KMS generates playlist players which do not have a 'playlist' tag, but something like 'playlist2.42', so searching for 'playlist' won't catch these cases.